resources/primary_zone.rb

```diff
@@ -28,7 +28,7 @@ def do_create(file_action)
       path "#{service_resource.vardir}/primary/db.#{new_resource.name}"
       owner service_resource.run_user
       group service_resource.run_group
-      mode 0o440
```

Comment:
This isn't a typo. According to the ruby style guide octal numbers should preferably be formatted in this manner. See https://github.com/rubocop-hq/ruby-style-guide#numeric-literal-prefixes for details.